src/app/core/config/app.json

```diff
@@ -218,6 +218,13 @@
       "path": "/verify-email/:token",
       "exact": false
     },
+    {
+      "id": "request-access",
+      "layout": "empty",
+      "path": "/request-access",
```

Comment:
Does the url not have to have a file or folder id?

Reply:
I don't know how this is going to work, we will have to take it up again when the back is finished and we start with the logic.